composer.json

```diff
@@ -77,7 +77,7 @@
     "piwik/device-detector": "^3.9",
     "presta/sitemap-bundle": "^1.7 || ^2.1",
     "ramsey/uuid": "^3.8",
-    "sabre/dav": "^3.2",
+    "sabre/dav": "^4.1.1",
```

Comment:
As this is out of context, I'd revert this here as well.

Reply:
Just moved that to #7511

Reply:
Is because of https://github.com/sabre-io/dav/pull/1269
It will fix an phpstan error. But I can move it out.

Reply:
Ok I see, you moved it out :)

Reply:
yes, please, I'd love to have this breaking change documented 😉 
Then it's easier for us to track all breaking changes and make a proper upgrade guide 😉